core/commands/add.go

```diff
@@ -36,6 +36,26 @@ Adds contents of <path> to ipfs. Use -r to add directories.
 Note that directories are added recursively, to form the ipfs
 MerkleDAG.
 `,
+		LongDescription: `
+Adds contents of <path> to ipfs. Use -r to add directories.
+Note that directories are added recursively, to form the ipfs
+MerkleDAG.
+
+Note that using the wrap option, '-w', wraps the file (or files,
```

Comment:
Jeromy Johnson <reply@reply.github.com> / 2016-03-17 17:49:11-0700
> > @@ -36,6 +36,26 @@ Adds contents of <path> to ipfs. Use -r to add directories.
> >  Note that directories are added recursively, to form the ipfs
> >  MerkleDAG.
> >  `,
> > +		LongDescription: `
> > +Adds contents of <path> to ipfs. Use -r to add directories.
> > +Note that directories are added recursively, to form the ipfs
> > +MerkleDAG.
> > +
> > +Note that using the wrap option, '-w', wraps the file (or files,
> 
> 'Also note...'  maybe? It sounds weird that we say "note that" to start two sentences in a row
> 

Are the "Note that"s even necessary?

Also:
• nix this comma: s/y, t/y t/g
• is it ipfs or IPFS?
• is <path> contents added to IPFS or the local repo?---I can add
  without a daemon running.